<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>t_total</class>
 <widget class="QWidget" name="t_total">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>741</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{border-image: url(:/image/picture/功能选择界面.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>修改题目</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>添加题目</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>用户</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>名单</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>专业设置</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/picture/修改按钮-橙色.png);</string>
          </property>
          <property name="text">
           <string>章节设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
